Boot2Docker Status/AppDelegate.m

```diff
@@ -172,11 +180,24 @@ - (void)stopBoot2Docker {
     
     [task setTerminationHandler:^(NSTask *t) {
         _haveRequestedStateChange = NO;
-        _statusItem.image = [NSImage imageNamed:@"docker-alt"];
+		_statusItem.image = [self _iconWithNotificationBubbleWithColor:[NSColor redColor]];
     }];
     
     [task launch];
 }
 
+- (NSImage*)_iconWithNotificationBubbleWithColor:(NSColor*)color {
+	NSImage* image = [(_darkMode ? [NSImage imageNamed:@"docker-dark"] : [NSImage imageNamed:@"docker"]) copy];
+
```

Comment:
it gives a non-color-based indication of the state, red-green colorblind users will not be able to differentiate between the states without it.